pages/success-banner/index.js

```diff
@@ -1,13 +1,47 @@
 import React from 'react';
 import Head from 'next/head';
 
-class Button extends React.Component {
+const eventLog = (eventDesc) => {
+  const data = 'Banner ' + eventDesc;
+
+  return () => {
+    const url = new URL(location.href);
+
+    if (url.searchParams.get('etcher-version').indexOf('1.0.0') !== -1) {
+      console.log(data);
+
+    } else {
+      const logJSON = JSON.stringify({
+        command: 'log',
+        data
+      });
+
+      console.log(logJSON);
+    }
+  }
+}
+
+class Button extends React.PureComponent {
+  render() {
+    return (
+      <a
+        href={ this.props.href }
+        target="_blank"
+        className="button"
+        onClick={ eventLog(`click ${this.props.label} button`) }>
+        { this.props.children }
+      </a>
+    )
+  }
+}
+
+class Link extends React.PureComponent {
```

Comment:
Given that the `Button` and `Link` here have `target="_blank"` I wonder if they should be named e.g. `ExternalButton` and `ExternalLink` to emphasise the fact that they'll open a new browser window, rather than being displayed inside the Etcher 'iframe' ?
Or perhaps it makes more sense to leave them as-is (i.e. open-externally by default), and then if we ever wanted Buttons or Links that opened *inside* the Etcher webview, we could name them e.g. `InternalButton` and `InternalLink`?